components/webui/.meteor/versions

```diff
@@ -4,16 +4,16 @@ babel-compiler@7.10.5
 babel-runtime@1.5.1
```

Comment:
Note this file was auto-updated by `meteor upgrade` after the packages specification changes in `.meteor/packages`.